test/checkFileIsUploadedFromUrl.spec.ts

```diff
@@ -2,23 +2,31 @@ import * as factory from './_fixtureFactory'
 import checkFileIsUploadedFromUrl from '../src/checkFileIsUploadedFromUrl'
 import {StatusEnum} from '../src/api/fromUrlStatus'
 import {getSettingsForTesting} from './_helpers'
+import fromUrl from '../src/api/fromUrl'
 
-describe('checkFileIsUploadedFromUrl', () => {
+describe('checkFileIsUploadedFromUrl', async() => {
```

Comment:
Seems that this function has become `async` by mistake. I haven't found `await` inside and maybe jasmine doesn't work well with async describes. (I'm not sure about that)

Reply:
Yes, you are right! Thank you